### ISSUE TITLE
Output errors?

### ISSUE THREAD

**Issue by RobLoach** [stevenschobert/metalsmith-sass#15]
When SASS building fails, it just outputs:

```
> metalsmith
  Metalsmith · undefined
```

We do return the error, should Metalsmith do a better job of error reporting?

**Comment by stevenschobert** [comment]
@RobLoach Were you still having issues with this?

**Comment by RobLoach** [comment]
@stevenschobert I believe I fixed the error with that pull request branch. Will do more testing and report back.

**Comment by RobLoach** [comment]
Syntax errors still report `undefined` from Metalsmith.

**Comment by stevenschobert** [comment]
@RobLoach How are you handling errors in your project? Any compilation errors should be available through Metalsmith's `build` callback (this was handled in #12). 

```js
metalsmith(__dirname)
  .use(sass())
  .build(function done(err) {
    // err should contain errors
  });
```

**Comment by madeofpeople** [comment]
@stevenschobert we're using metalsmith.json, is there a catch (or a plugin) that can gracefully handle errors in a similar manner?

**Comment by stevenschobert** [comment]
Corrected in [v0.6.1](https://github.com/stevenschobert/metalsmith-sass/releases/tag/v0.6.1)! Sorry about that folks, looks like when it was implemented in #12, errors were being passed up the chain as a simple string.

Metalsmith's CLI expects the error to be an [actual Error object](https://github.com/segmentio/metalsmith/blob/86c0223a0d494468a74f0d782fedfb529e94e7f8/bin/metalsmith#L97-L100). I've updated the tests and patched it in 9a1586109bcd888a543015209b47ae07b3026f01.

Let me know if that works for you!

**Comment by RobLoach** [comment]
![](http://24.media.tumblr.com/tumblr_mc9lrzeC0a1qkab13o1_500.gif)
